middlewares/logger.js

```diff
@@ -10,7 +10,7 @@ const loggerConfig = config.get('logger');
  * @returns A transport for daily rotate file
  */
 const dailyRotateFileTransport = new (winston.transports.DailyRotateFile)({
-  filename: 'express-api-skeleton-%DATE%.log',
```

Comment:
Instead of modifying this manually, I made a change in the skeleton to dynamically creating log file based on each API name: https://github.com/osu-mist/express-api-skeleton/pull/26/files#diff-c128155a1e5a44014a4c3501e4bd7e8bR16

Can you bring the change from that PR?